BuildingBlocks/src/BuildingBlocks.Application/MediatR/LoggingBehavior.cs

```diff
@@ -35,9 +36,31 @@ private void Before()
     private void After()
     {
         _watch!.Stop();
+        _logger.HandleRequest(typeof(TRequest).Name, _watch.ElapsedMilliseconds);
+    }
+}
 
-        var logLevel = _watch.ElapsedMilliseconds < 1000 ? LogLevel.Information : LogLevel.Warning;
+file static class LoggerExtensions
+{
+    private static readonly Action<ILogger, string, long, Exception> HANDLED_REQUEST_WARNING =
+        LoggerMessage.Define<string, long>(
+            LogLevel.Warning,
+            new EventId(437002, "LoggingBehavior.HandleRequestInformation"),
```

Comment:
The name doesn't have to contain the log level. It can be the same for HANDLED_REQUEST_WARNING and HANDLED_REQUEST_INFORMATION.